<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QGroupBox" name="ScopeSettings">
      <property name="title">
       <string>Scope Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="SampleSettings">
         <property name="title">
          <string>Sample / Buffer Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Samples</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="samplesSpinBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Records per Buffer</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="recordsperbufferSpinBox"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Buffers</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="buffersSpinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="VerticalSettings">
         <property name="title">
          <string>Vertical Settings</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="Ch1Vertical">
            <property name="title">
             <string>Channel 1</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QFormLayout" name="formLayout_5">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QCheckBox" name="ch1_enabledCheckBox">
                 <property name="text">
                  <string>Enabled</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="ch1_filteredCheckBox">
                 <property name="text">
                  <string>Filtered</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Range</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="ch1_rangeComboBox">
                 <item>
                  <property name="text">
                   <string>4 V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2 V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1 V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>400 mV</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>200 mV</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>100 mV</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>40 mV</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Coupling</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="ch1_couplingComboBox">
                 <item>
                  <property name="text">
                   <string>DC</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>AC</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="Ch2Vertical">
            <property name="title">
             <string>Channel 2</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QFormLayout" name="formLayout_6">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QCheckBox" name="ch2_enabledCheckBox">
                 <property name="text">
                  <string>Enabled</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="ch2_filteredCheckBox">
                 <property name="text">
                  <string>Filtered</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Range</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="ch2_rangeComboBox">
                 <item>
                  <property name="text">
                   <string>4 V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2 V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1 V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>400 mV</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>200 mV</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>100 mV</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>40 mV</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Coupling</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="ch2_couplingComboBox">
                 <item>
                  <property name="text">
                   <string>DC</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>AC</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ClockSettings">
         <property name="title">
          <string>Clock Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Clock Rate</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="samplerateComboBox">
              <item>
               <property name="text">
                <string>1 GHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>500 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>250 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20 kHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10 kHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5 kHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2 kHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1 kHz</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Clock Source</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="clocksourceComboBox">
              <item>
               <property name="text">
                <string>Internal</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Reference</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>60 MHz</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1 GHz</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Clock Edge</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="clockedgeComboBox">
              <item>
               <property name="text">
                <string>Rising</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Falling</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Timeout</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="timeoutSpinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="TriggerSettings">
         <property name="title">
          <string>Trigger Settings</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Source 1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_4">
              <item>
               <property name="text">
                <string>CH 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CH 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>External</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Disabled</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_5">
              <item>
               <property name="text">
                <string>Rising</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Falling</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Level 1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="ch1_triglevelSpinBox"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Coupling</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="comboBox_7">
              <item>
               <property name="text">
                <string>DC</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>AC</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Edge 1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Source 1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_6">
              <item>
               <property name="text">
                <string>CH 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>CH 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>External</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Disabled</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBox_8">
              <item>
               <property name="text">
                <string>Rising</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Falling</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Level 1</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="ch1_triglevelSpinBox_2"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Coupling</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="comboBox_9">
              <item>
               <property name="text">
                <string>DC</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>AC</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Edge 1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="datapathButton">
          <property name="text">
           <string>Datapath</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autorunCheckBox">
          <property name="text">
           <string>autorun</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="x_autoscaleCheckBox">
          <property name="text">
           <string>autoscale X</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="y_autoscaleCheckBox">
          <property name="text">
           <string>autoscale Y</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CurveWidget" name="curvewidget">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CurveWidget</class>
   <extends>QSplitter</extends>
   <header>guiqwt.plot</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
